### DISCUSSION TITLE
How can I connect a table to a scatter plot?

### DISCUSSION THREAD

**Discussion by falconair** [vega/altair#2955] [Q&A]
I'm working in Jupyter lab and need to do, what I thought was, fairly simple: when a user clicks on a point in s scatter plot, display the underlying data row in a table. Perhaps filter an existing dataframe table via columns behind the clicked point. 

I imagine this will require some sort of ipywidget connect? I'm having a terrible time finding how to do this.

There are a couple of examples on altair's website and holoviews websites (links below), but they give me errors related to param.brush not being found or something about checking javascript console for more details.

https://altair-viz.github.io/gallery/scatter_linked_table.html
https://panel.holoviz.org/gallery/simple/altair_brushing.html

Does someone have a pre-written example of this feature?

**Comment by joelostblom** [comment]
The altair docs example you linked works fine for me. Could you be more specific about the error you are encountering? Here is a slightly modified version of that example https://stackoverflow.com/questions/67997825/python-altair-generate-a-table-on-selection/68005515#68005515
